test/test_wallet.rb

```diff
@@ -122,4 +122,16 @@ def test_iterates_income_transactions
       )
     end
   end
+
+  def test_not_allow_amount_exceed_max
+    coins = 2**63 - 1
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      time = Time.now
+      wallet.add(Zold::Txn.new(1, time, Zold::Amount.new(coins: 2), 'NOPREFIX', Zold::Id.new, '-'))
+      assert_raises RuntimeError do
```

Comment:
@liligga check if the expected error was raised. See `test_source.rb`  as an example.

Reply:
Thank you. Messed the things up, sorry. However I couldn't find `test_source.rb` though.

Reply:
@liligga Sorry, this file is in another repository. See [here](https://github.com/yegor256/pdd/blob/8c7f298626b2fcd200997372fd19d5c72819fa9d/test/test_source.rb#L67-L70).